mscore/musescore.cpp

```diff
@@ -6210,7 +6280,6 @@ int main(int argc, char* av[])
             mscore->showSynthControl(true);
       if (settings.value("mixerVisible", false).toBool())
             mscore->showMixer(true);
-
```

Comment:
There is a method to get this done automatically for Qt Creator, but I don't know whether XCode has this too